### DISCUSSION TITLE
Gaussian field from ellipsoid

### DISCUSSION THREAD

**Discussion by jonathan-taylor** [GeoStat-Framework/GSTools#322] [Q&A]
I'd like to create a `Gaussian` instance in either 2 or 3 (or perhaps 4) dim starting directly from the quadratic form Q (i.e. my correlation is cor(h)=exp(-h @ Q @ h)). Obviously this is determined by U, D in the eigen-decomposition Q= U @ D @ U.T and I realize that U, D will be used in the `anis` and `angles` argument but I'm uncertain what the mapping would be as I'm not too familiar with Tait-Bryan angles.

**Comment by LSchueler** [comment]
Hi jonathan-tayler,

I'm sorry for the late reply.
I guess you could use some other Python packages to do the work for you. Did you have a look at the [transform](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.transform.Rotation.html) parts of scipy or the [transforms3d](https://github.com/matthew-brett/transforms3d) package?
Using something like that should make it pretty easy to convert between your representation of the rotation and GSTools' internal one.
